test/test_suite.bats

```diff
@@ -300,7 +300,16 @@
   [[ ${lines[0]} == "-rw-r--r--"* ]]
 }
 
-@test "Final part of the tests: Kill pihole-FTL process" {
-  run bash -c 'kill $(pidof pihole-FTL)'
+@test "Dependence on shared libraries" {
+  run bash -c 'ldd ./pihole-FTL'
   printf "%s\n" "${lines[@]}"
+  [[ "${STATIC}" != "true" && "${lines[@]}" == *"=>"* ]] || \
```

Comment:
Please add a comment to explain the significance of checking for this string, and do the same for the other test.